examples/iga/collocation_stokes_problem_sparse.py

```
@@ -0,0 +1,444 @@
+# Analytical solutions
+
+
+def analytical_solution_u(geometry, on):
+    x_vec = geometry.evaluate(on)
+    x = x_vec[:, 0]
+    y = x_vec[:, 1]
+    return (x**2) * (1.0 - x) * (1.0 - x) * (2 * y - 6 * y**2 + 4 * y**3)
+
+
+def error_u(solution_u):
+    def error_u(data, on):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `error_u`

(ANN202)